core/kernel/notif_default.c

```diff
@@ -1,97 +1,168 @@
 // SPDX-License-Identifier: BSD-2-Clause
 /*
- * Copyright (c) 2021-2023, Linaro Limited
+ * Copyright (c) 2021-2024, Linaro Limited
  */
 
 #include <assert.h>
```

Comment:
Add `#include <config.h>` (for `IS_ENABLED()`)

Reply:
Noted.